packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4001,6 +4001,31 @@ class IndexedStack extends Stack {
       ..alignment = alignment
       ..textDirection = textDirection ?? Directionality.maybeOf(context);
   }
+
+  @override
+  MultiChildRenderObjectElement createElement() {
+    return _IndexedStackElement(this);
+  }
+}
+
+class _IndexedStackElement extends MultiChildRenderObjectElement {
+  _IndexedStackElement(IndexedStack super.widget);
+
+  @override
+  IndexedStack get widget => super.widget as IndexedStack;
+
+  @override
+  void debugVisitOnstageChildren(ElementVisitor visitor) {
+    final int? index = widget.index;
+    if (index == null) {
+      return super.debugVisitOnstageChildren(visitor);
```

Comment:
This will consider all children onstage, right? I though if the index is null, the IndexedStack displays nothing, so all of them should be offstage?